cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,111 @@
+    # Check if the origin is allowed
+    if origin not in ALLOWED_ORIGINS:
+        return {'error': 'Forbidden - Origin not allowed'}, 403
+
+    # Add CORS headers to the response
+    headers = {
+        'Access-Control-Allow-Origin': origin,
+        'Access-Control-Allow-Methods': 'POST, OPTIONS',
+        'Access-Control-Allow-Headers': 'Authorization, Content-Type',
+    }
+
+    return headers
```

Ruff (0.8.2)

Unnecessary assignment to `headers` before `return` statement

Remove unnecessary assignment

(RET504)